### PR TITLE
Metadata

### PR DESCRIPTION
**Pull request by alina-zaman**
This change is [Reviewable](https://reviewable.io/reviews/md2korg/cerebralcortex-dataanalysis/72)

### CODE REVIEW

File: core/resources/metadata/arrival_time.json [hunk 1]
```diff
@@ -2,8 +2,20 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"list",
-      "unit":""
+      "name": "office_arrival_time",
+      "type":"datetime",
```
Comment (twhnat) [inline core/resources/metadata/arrival_time.json]: What is the data type you are saving here?  Datetime is likely not the correct thing to store.

File: core/resources/metadata/arrival_time_from_beacon.json [hunk 2]
```diff
@@ -2,8 +2,20 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"list",
-      "unit":""
+      "name": "arrival_time_around_beacon",
+      "type":"datetime",
```
Comment (twhnat) [inline core/resources/metadata/arrival_time_from_beacon.json]: What is the data type you are saving here?  Datetime is likely not the correct thing to store.

File: core/resources/metadata/expected_conservative_arrival_time.json [hunk 3]
```diff
@@ -2,8 +2,20 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"list",
-      "unit":""
+      "name": "office_arrival_time",
+      "type":"datetime",
```
Comment (twhnat) [inline core/resources/metadata/expected_conservative_arrival_time.json]: What is the data type you are saving here?  Datetime is likely not the correct thing to store.

File: core/resources/metadata/expected_conservative_arrival_time_from_beacon.json [hunk 4]
```diff
@@ -2,8 +2,20 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"list",
-      "unit":""
+      "name": "arrival_time_around_beacon",
+      "type":"datetime",
```
Comment (twhnat) [inline core/resources/metadata/expected_conservative_arrival_time_from_beacon.json]: What is the data type you are saving here?  Datetime is likely not the correct thing to store.

File: core/resources/metadata/expected_liberal_arrival_time.json [hunk 5]
```diff
@@ -2,8 +2,20 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"list",
-      "unit":""
+      "name": "office_arrival_time",
+      "type":"datetime",
```
Comment (twhnat) [inline core/resources/metadata/expected_liberal_arrival_time.json]: What is the data type you are saving here?  Datetime is likely not the correct thing to store.

File: core/resources/metadata/expected_liberal_arrival_time_from_beacon.json [hunk 6]
```diff
@@ -2,8 +2,20 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"list",
-      "unit":""
+      "name": "arrival_time_around_beacon",
+      "type":"datetime",
```
Comment (twhnat) [inline core/resources/metadata/expected_liberal_arrival_time_from_beacon.json]: What is the data type you are saving here?  Datetime is likely not the correct thing to store.

File: core/resources/metadata/working_days.json [hunk 7]
```diff
@@ -2,8 +2,9 @@
   "annotations":[],
   "data_descriptor":[
     {
+      "name": "office_day's_annotation_from_gps_location_model",
       "type":"String",
-      "unit":""
+      "description": "Office"
```
Comment (twhnat) [inline core/resources/metadata/working_days.json]: Details

File: core/resources/metadata/working_days_from_beacon.json [hunk 8]
```diff
@@ -2,8 +2,9 @@
   "annotations":[],
   "data_descriptor":[
     {
-      "type":"string",
-      "unit":""
+      "name": "office_day's_annotation_from_beacon_model",
+      "type":"String",
+      "description": "work"
```
Comment (twhnat) [inline core/resources/metadata/working_days_from_beacon.json]: Details

**Comment by twhnat** [comment]
You also need to ensure that your PR can be merged easily.  E.g. you need to resolved any merge conflicts.

**Comment by alina-zaman** [comment]
Dr, Hnat,

I am trying to resolve the conflicts. "

    "type":"list",
-      "unit":""

"

These two  line is from the previous code, shouldn't be there. I am trying
to find a way so that the conflicts do not occur.
Sorry for the inconvenience. May be you can understand this the last few
days of semester. I will try to solve this by today.

Alina

**Comment by alina-zaman** [comment]
Dr. Hnat
I have resolved the conflicts. Could you please look into it?

**Comment by twhnat** [comment]
Merged.

**Pull request merged**